portal-test/src/com/liferay/portal/kernel/test/util/RandomTestUtil.java

```diff
@@ -46,6 +47,20 @@ public static Timestamp nextTimestamp() {
 		return new Timestamp(nextDate().getTime());
 	}
 
+	public static String randomBase64BySize(int sizeInBytes) {
```

Comment:
Do not add a util method that is only used once. Inline this logic in the test.

Reply:
@ZakiMiklos I agree about inlining. Also, I'd consider whether to add this new object field in the setup, or in the test method? It's only used in a single test only, but will be now be hanging around in every other one as well. Because this test class is huge in many ways already, I think we should not add extra overhead to the other tests.